data-raw/qc-markdowns/adult-upstream-passage-monitoring/deer-creek/extract_deer_upstream_estimates_from_raw_files.R

```diff
@@ -0,0 +1,144 @@
+# code to extract adult passage estimates from raw deer creek files
+# estimates were manually extracted into `deer_estimates_manual_extract.xlsx`
+
+# code here is only helpful for automating/extracting
+
+# Deer Creek and Mill Creek files were sent individually from Doug Killam and added to the google bucket
+
```

Comment:
if code is being used we should add these:

`library(tidyverse)`
`library(googleCloudStorageR)`
`library(here)`